iOS/DuckDuckGo/MainViewController.swift

```diff
@@ -963,7 +963,12 @@ class MainViewController: UIViewController {
         }
     }
 
-    func loadQueryInNewTab(_ query: String, reuseExisting: Bool = false) {
+    enum ReuseExisting: Equatable {
```

Comment:
Given the scope this is defined at, I think it could do with a more specific name

Reply:
Nice, that's much better!